src/app/static/src/app/components/files/ManageFile.vue

```diff
@@ -2,6 +2,7 @@
     <div class="form-group">
         <label class="font-weight-bold mb-0" v-translate="label"></label>
         <tick color="#e31837" v-if="valid" width="20px"></tick>
+        <a class="mb-0" v-if="hasValidAdr" href="#">ADR</a>
```

Comment:
This isn't actually a link, should just be a `span` tag (with `.color-red` for styling)

Reply:
I initially thought this would eventually be a link back to the ADR as seen in comment by Rich on the mockup. It is now changed to span.